processes/local_development/seed_test_data.py

```diff
@@ -47,7 +47,15 @@ def runProcess(self):
             self.db_manager.close_connection()
 
     def save_test_record(self):
-        test_record = Record(**self.test_data)
-
-        self.db_manager.session.add(test_record)
-        self.db_manager.session.commit()
+        existing_record = self.db_manager.session.query(Record).filter_by(uuid=self.test_data['uuid']).first()
```

Comment:
The test_data['uuid'] is going to change every time so existing record will always be None. 

I recommend using source_id which will remain constant